taxi/tests/test_forms.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTestCase(TestCase):
+    def test_driver_creation_form_with_license_number_first_last_name(self):
+        test_data = {
+            "username": "test_user",
+            "license_number": "NNN12345",
+            "first_name": "John",
+            "last_name": "Doe",
+            "password1": "<PASSWORD>",
+        }
+
+        form = DriverCreationForm(data=test_data)
+        self.assertFalse(form.is_valid())
```

Comment:
The test currently expects the form to be invalid (`assertFalse(form.is_valid())`), but the provided data seems correct. If the form should be valid, change this to `assertTrue(form.is_valid())`.